src/common/transformations/src/transformations/op_conversions/convert_xor_to_logical_xor.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_xor_to_logical_xor.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <openvino/opsets/opset1.hpp>
+
+#include "itt.hpp"
+
+ov::pass::ConvertXorToLogicalXor::ConvertXorToLogicalXor() {
+    MATCHER_SCOPE(ConvertXorToLogicalXor);
+
+    auto xor_v1 = pattern::wrap_type<ov::opset1::Xor>();
+
+    matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto xor_v1_node = std::dynamic_pointer_cast<ov::opset1::Xor>(m.get_match_root());
+        if (!xor_v1_node)
+            return false;
+
+        const auto& autobroad = xor_v1_node->get_autob();
+
+        auto logical_xor_v1 = std::make_shared<ov::opset1::LogicalXor>(xor_v1_node->input_value(0),
```

Comment:
nitpick: it's better to use the latest opset (opset10) version for LogicalXor

Reply:
done